projects/manage-pom-cases-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/PomCaseMessageHandler.kt

```diff
@@ -15,23 +22,76 @@ import uk.gov.justice.digital.hmpps.telemetry.notificationReceived
 @Component
 @Channel("manage-pom-cases-and-delius-queue")
 class PomCaseMessageHandler(
-    override val converter: NotificationConverter<HmppsDomainEvent>,
+    @Value("\${mpc.handover.url}") private val mpcHandoverUrl: String,
+    override val converter: PomCaseEventConverter,
     private val handoverDatesChanged: HandoverDatesChanged,
     private val pomAllocated: PomAllocated,
-    private val telemetryService: TelemetryService
-) : NotificationHandler<HmppsDomainEvent> {
+    private val telemetryService: TelemetryService,
+    private val personRepository: PersonRepository
+) : NotificationHandler<Any> {
     @Publish(
         messages = [
             Message(name = "offender-management/handover-changed"),
             Message(name = "offender-management/pom-allocated")
```

Comment:
Could we add the AsyncAPI doc for the new message type?
```suggestion
            Message(name = "offender-management/pom-allocated"),
            Message(messageId = "SENTENCE_CHANGED", payload = Schema(ProbationOffenderEvent::class)),
```

Reply:
ahh sorry about that!